### ISSUE TITLE
token generation outdated

### ISSUE THREAD

**Issue by Randomneo** [phistrom/basecampy3#4]
im getting error 
```
Traceback (most recent call last):
  File "/usr/local/bin/bc3", line 11, in <module>
    load_entry_point('basecampy3==0.2.2', 'console_scripts', 'bc3')()
  File "/usr/local/lib/python2.7/dist-packages/basecampy3/bc3_cli.py", line 122, in main
    return CLI.from_command_line()
  File "/usr/local/lib/python2.7/dist-packages/basecampy3/bc3_cli.py", line 38, in from_command_line
    args.func()
  File "/usr/local/lib/python2.7/dist-packages/basecampy3/bc3_cli.py", line 61, in _configure
    code = requestor.get_user_code()
  File "/usr/local/lib/python2.7/dist-packages/basecampy3/token_requestor.py", line 53, in get_user_code
    user_code = self._submit_authorize_app_form(login_resp.url, login_resp.text)
  File "/usr/local/lib/python2.7/dist-packages/basecampy3/token_requestor.py", line 100, in _submit_authorize_app_form
    auth_code = parse_qs(urlparse(app_redirect_uri).query)['code'][0]
KeyError: 'code'
```
I printed response and get google.api 
```
https://accounts.google.com/o/oauth2/auth?client_id=341981671303-g568phroodi4guli3inqb7gnsvbvqnlk.apps.googleusercontent.com&prompt=login&redirect_uri=https%3A%2F%2Flaunchpad.37signals.com%2Fgoogle_sign_in%2Fcallback&response_type=code&scope=openid+profile+email&state=SyLv%2Bvf5g11LVk%2BGMlNWR1psf7Q%2Bnby5
```

**Comment by brianredbeard** [comment]
Hit the same issue.  It's because there are multiple forms on the auth page.  When the call is made through BeautifulSoup to grab the form:

https://github.com/phistrom/basecampy3/blob/f4ad11ceb3273a2b2b8cc88ac0a2f40805818882/basecampy3/token_requestor.py#L86-L93

It ends up grabbing the first form on the page, which is the Google auth login.

A couple of print statements right after that shows the following:

```
url:  https://launchpad.37signals.com/signin
auth_form_url:  /google_sign_in/authorization
```

**Comment by phistrom** [comment]
This is now resolved in 0.3.0. **Username and password can no longer be specified** in `bc3 configure`. Instead the CLI opens a browser window to an authorization page at Basecamp's parent company "37signals." You must sign in using your browser, and choose to allow the integration you created yourself. **Your redirect URI should be set to http://localhost:33333** so that the CLI, which will start a simple HTTP server on that port, can receive the authorization code from your browser and complete the process of obtaining the access and refresh tokens.

If you want to implement BasecamPY3 on a computer with no GUI (with no web browser), you can complete the process on a computer that does, and copy over the resulting basecamp.conf to the target server. From there, the refresh token should allow it to obtain new access tokens and you will not have to do this process again.
